daemon/cmd/daemon_main.go

```diff
@@ -335,7 +335,10 @@ func init() {
 	flags.String(option.EgressMasqueradeInterfaces, "", "Limit egress masquerading to interface selector")
 	option.BindEnv(option.EgressMasqueradeInterfaces)
 
-	flags.Bool(option.EnableHostReachableServices, false, "Enable reachability of services for host applications")
+	flags.Bool(option.SkipSocketLBInPodNamespace, false, "Skip socket LB for services when inside a pod namespace, in favor of service LB at the pod interface. Socket LB is still used when in the host namespace.")
+	option.BindEnv(option.SkipSocketLBInPodNamespace)
+
+	flags.Bool(option.EnableHostReachableServices, false, "Enable reachability of services for host applications (beta)")
```

Comment:
Where did the beta come from? This feature has been marked non-beta since 714a56e67425e0e53a30737dd22648eb9b21e8ed (v1.8.0).

Reply:
oops, must messed it up during rebase or something...fixed.